cffi/cdefs.h

```diff
@@ -434,6 +437,21 @@ struct lyd_node_leaf_list {
 	...;
 };
 
+typedef enum {
+    LYD_DIFF_END = 0,
+    LYD_DIFF_DELETED,
+    LYD_DIFF_CHANGED,
+    LYD_DIFF_MOVEDAFTER1,
+    LYD_DIFF_CREATED,
+    LYD_DIFF_MOVEDAFTER2
+} LYD_DIFFTYPE;
+
+struct lyd_difflist {
+    LYD_DIFFTYPE *type;
+    struct lyd_node **first;
+    struct lyd_node **second;
```

Comment:
Maybe add an ellipsis `...` here as well, just in case new fields are added to `struct lyd_difflist`.